agent/runner.go

```diff
@@ -59,10 +98,10 @@ func (r *Runner) Run(runnerCtx context.Context) error {
 	// get the next workflow from the queue
 	work, err := r.client.Next(runnerCtx, r.filter)
```

Comment:
The server is blocking `Next` request until it has a job, so there should be no need to backoff here.